### DISCUSSION TITLE
Programa para leitura de imagens

### DISCUSSION THREAD

**Discussion by ceslima** [community/community#147884] [Programming Help]
### Body

Pessoal, preciso de ajuda para criar um programa que leia um encarte de supermercado e identifique os produtos e os preços do encarte e transforme as informações em uma planilha do Excel. por exemplo: CERVEJA BRAHMA 550ML -  R$ 4,99, a descrição em uma coluna e o preço em outra. cada produto lido ir acrescentando em cada linha. obrigado 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by miguelangelooscardoso** [comment]
Qual a pergunta mesmo? Queres saber a linguagem que deves usar ou? Eu faria em C, acho bem simples, contudo exista quem te indicará shell scripting ou até mesmo python.

**Comment by mecodeatlas** [comment]
Hi @ceslima, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
